src/metatrain/experimental/phace/modules/tensor_sum.py

```diff
@@ -0,0 +1,113 @@
+from typing import List, Tuple
+
+import metatensor.torch
+import torch
+from metatensor.torch import Labels
+
+from .layers import EquivariantLinear
+
+
+class EquivariantTensorAdd(torch.nn.Module):
```

Comment:
Can you explain what this is doing? I could not quite understand from reading the code.